charts/cryostat/templates/cryostat_deployment.yaml

```diff
@@ -216,3 +222,8 @@ spec:
         secret:
           secretName: {{ .Release.Name }}-proxy-tls
       {{- end }}
+      {{- range .Values.core.config.eventTemplates.configMapNames}}
+      - name: {{ . }}
+        configMap:
+          name: {{ . }}
+      {{- end }}
```

Comment:
Any thoughts on using a projected volume here to combine all config-map contents into a single volume? One benefit is that we can define a single volumeMount on `/opt/cryostat.d/templates.d`, thus reducing the manifest/object def size.

Reference: https://kubernetes.io/docs/concepts/storage/projected-volumes/

Reply:
hm, I suppose reducing the number of volume mount definitions can help reduce the manifest size (thus, object size in etcd). And grouping them under a single volume seems to make it easier to recognize their purpose when reading the manifest (my opinion).

Reply:
I guess we can do the same for https://github.com/cryostatio/cryostat-helm/pull/219.